apollo-router/tests/snapshots/apollo_reports__client_name.snap

```diff
@@ -589,5 +589,5 @@ traces_per_query:
     internal_traces_contributing_to_stats: []
 end_time: "[end_time]"
 operation_count: 1
-traces_pre_aggregated: false
+traces_pre_aggregated: true
```

Comment:
A bit surprising not to see any tests where this affects the stats...

Reply:
Gonna add a new metrics integration test

Reply:
Added in [88cf26e](https://github.com/apollographql/router/pull/2277/commits/88cf26ee9fd6f70d000005ef0b505ebfd9588c1f) and [a21762b](https://github.com/apollographql/router/pull/2277/commits/a21762b37ea9272d3a54cedf646885b35afa9b26)